paddle/fluid/pybind/ir.cc

```diff
@@ -21,391 +21,51 @@
-    setter(attr_name, attr_value, pass);
-  }
-
- private:
-  std::unordered_map<std::string, GetterSetter> getter_setter_map_;
-};
-
-#define REGISTER_PASS_ATTR_GETTER_SETTER(attr_type_name, cpp_type)             \
-  do {                                                                         \
-    auto getter = [](const framework::ir::Pass &pass,                          \
-                     const std::string &attr_name) -> py::object {             \
-      auto attr_value = pass.Get<cpp_type>(attr_name);                         \
-      return py::cast(attr_value);                                             \
-    };                                                                         \
-    auto setter = [](const std::string &attr_name,                             \
-                     const py::object &attr_value,                             \
-                     framework::ir::Pass *pass) {                              \
-      PADDLE_ENFORCE_NOT_NULL(                                                 \
-          pass, platform::errors::InvalidArgument("pass should be provided")); \
-      try {                                                                    \
-        const auto &cpp_attr_value = py::cast<cpp_type>(attr_value);           \
-        pass->Set(attr_name, new cpp_type(cpp_attr_value));                    \
-      } catch (py::cast_error &) {                                             \
-        PADDLE_THROW(platform::errors::InvalidArgument(                        \
-            "type error of attribute %s, expected to be %s",                   \
-            attr_name,                                                         \
-            attr_type_name));                                                  \
-      }                                                                        \
-    };                                                                         \
-    PassAttrGetterSetterRegistry::Instance().Register(                         \
-        attr_type_name, getter, setter);                                       \
-  } while (0)
-
-// NOTE: attr_types may be changed
-static void SetAttrsToPass(
-    const std::unordered_map<std::string, py::object> &attrs,
-    std::unordered_map<std::string, std::string> *attr_types,
-    framework::ir::Pass *pass) {
-  for (const auto &name_and_value : attrs) {
-    const auto &attr_name = name_and_value.first;
-    const auto &attr_value = name_and_value.second;
-    auto &attr_type = (*attr_types)[attr_name];
-    if (attr_type.empty()) {
-      attr_type = py::cast<std::string>(attr_value.get_type().attr("__name__"));
-    }
-    PassAttrGetterSetterRegistry::Instance().Set(
-        attr_name, attr_type, attr_value, pass);
-  }
+void BindBlock(py::module *m) {
+  py::class_<Block> block(*m, "Block");
+  block.def("front", &Block::front, return_value_policy::reference)
+      .def("get_op_list", [](Block &self) -> py::list {
```

Comment:
```suggestion
      .def_property("ops", [](Block &self) -> py::list {
```
因为在`framework.py` 的旧体系下使用方式是 `block.ops`

Reply:
好的，同上